libcontainerd/client_daemon.go

```diff
@@ -266,15 +266,14 @@ func (c *client) Exec(ctx context.Context, containerID, processID string, spec *
 		stdinCloseSync = make(chan struct{})
 	)
 
-	fifos := newFIFOSet(ctr.bundleDir, containerID, processID, withStdin, spec.Terminal)
+	fifos := newFIFOSet(ctr.bundleDir, processID, withStdin, spec.Terminal)
 
 	defer func() {
 		if err != nil {
 			if rio != nil {
 				rio.Cancel()
 				rio.Close()
 			}
-			rmFIFOSet(fifos)
```

Comment:
This is now handled automatically by `DirectIO.Close()` which is called above (`rio.Close()`)